docs/conf.py

```diff
@@ -34,7 +34,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = [
+_EXTENSIONS = [
```

Comment:
```suggestion
extensions = [
```

Reply:
This is also used by Sphinx for extensions

Reply:
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-extensions